cli/cache/caches.rs

```diff
@@ -3,20 +3,21 @@
 use std::path::PathBuf;
 use std::sync::Arc;
 
+use deno_lib::cache::DenoDirProvider;
```

Comment:
I started shifting down some of this code shared by denort and deno, but then switched to targeting the worker.